README.md

```diff
@@ -50,8 +64,17 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
+```
+var pizza = {
+    temp : 70,
+    function bake() {
+        for (var i = 0; i <= 300; i++) {
+        i = temp;
+        }
+    }
+    bake();
+}
+
```

Comment:
I think you're thinking too hard about this one!  I don't think a for loop is right tool for this. All that it asks for in the method is that it resets the temperature to 300 when back is called.

```js
var pizza = {
  temp : 70,
  bake: function(){
    this.temp = 300
  }
 }
```